builtin_metadata.json

```diff
@@ -18256,6 +18257,26 @@
     },
     "wasm": false
   },
+  "uuid.parse": {
+    "args": [
+      {
+        "description": "uuid string to be evaluated",
+        "name": "uuid",
+        "type": "string"
+      }
+    ],
+    "available": [
+      "edge"
+    ],
+    "description": "Parses incoming string if valid uuid, returns object with properties if valid",
```

Comment:
Suggestion: "Parses the string value as an UUID and returns an object with the well-defined fields of the UUID if valid".
This suggestion also applies to the `description` in `var UUIDParse`

Reply:
All these are fair and, should be changed into something that would look better on the webpage. And also explained the different output variables. Think these were generated with `make generate`, so thats why the desc in `var UUIDParse` is similiar

Reply:
changed it into your suggestion and made it look more like other builtin docs